src/core/Table/TableRowMemoized.tsx

```diff
@@ -17,14 +18,15 @@ import { CSSTransition } from 'react-transition-group';
  */
 const TableRow = <T extends Record<string, unknown>>(props: {
   row: Row<T>;
-  rowProps: TableRowProps;
+  rowProps?: (rowData: T) => React.ComponentPropsWithRef<'div'>;
```

Comment:
Now, I wonder if we should also expose `row: Row<T>` instead of just rowData

Reply:
Changed.